### ISSUE TITLE
"Hide/Unhide" button of Password field present in "Connect to SQL Server" does not have name defined: A11y_mssql for VSCode_Connections_Name Role Value

### ISSUE THREAD

**Issue by ManishaC1** [microsoft/vscode-mssql#18655]
**Github Tags:** #A11TCS; #A11yMAS; #WCAG4.1.2; #Name Role Value; #FTP; #Win32; #Win11; #DesktopApp; #MSSQL for VSCode; #BM_MSSQL for VSCode_Win32_Feb2025; #AILimited; #Benchmark;

**Please do not close this bug. This bug should only be closed by Trusted Tester after verification** 

"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

## Environment Details:

Application Name: MSSQL for VSCode
Version: 1.98.0-insider (user setup)
Commit: fe4d6a831a4f87126187ac6ba96e16405d69de0f
Date: 2025-02-07T05:04:37.288Z
Electron: 32.2.7
ElectronBuildId: 10660205
Chromium: 128.0.6613.186
Node.js: 20.18.1
V8: 12.8.374.38-electron.0
OS: Windows_NT x64 10.0.26100

Windows 11 24H2 OS Build 26100.3037

## Actual Results:
"Hide/Unhide" button of Password field present in "Connect to SQL Server" dialog does not have a defined name.

## Expected Results:
"Hide/Unhide" button of Password field present in "Connect to SQL Server" dialog should have a defined name.

## User Impact:
Screen Reader Users cannot identify the "Hide/Unhide Password" button, leading to a poor user experience.

## MAS Reference:  

https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html

**Comment by ManishaC1** [comment]
Issue fixed in v1.34

![Image](https://github.com/user-attachments/assets/8f07d024-256d-4bd6-9e61-e91c94d4d56d)
